python/wipe_appwrite_bucket/main.py

```diff
@@ -0,0 +1,41 @@
+
+import json
+from appwrite.client import Client
+from appwrite.services.storage import Storage
+
+
+def main(req, res):
+    client = Client()
+    
+
+    print("Starting function")
+
+    # Make sure we have envirnment variables required to execute
+    if not req.variables.get('APPWRITE_FUNCTION_ENDPOINT', None) or not req.variables.get('APPWRITE_FUNCTION_API_KEY', None):
+        return res.json({"success":False,"message":"Required environment variables not found"})
+    
+    payload = json.loads(req.payload)
+
+    if 'bucketId' not in payload:
+        return res.json({"success":False,"message":"Missing bucketId"})
+
+    bucket_id = payload['bucketId']
+
+    (client
+        .set_endpoint(req.variables.get('APPWRITE_FUNCTION_ENDPOINT', None))
+        .set_project(req.variables.get('APPWRITE_FUNCTION_PROJECT_ID', None))
+        .set_key(req.variables.get('APPWRITE_FUNCTION_API_KEY', None))
+    )
+    storage = Storage(client)
+
+    # Get all files in the bucket
+    try:
+        files = storage.list_files(bucket_id)
+    except:
+        return res.json({"success":False,"message":"Bucket not found."})
+
+    # Delete all files in the bucket
+    for file in files['files']:
+        storage.delete_file(bucket_id,file['$id'])
```

Comment:
Because, by default, only a maximum of 25 files will be returned, this won't actually wipe the entire bucket. I recommend wrapping the whole block with a loop and break if `files` is empty.